rand_distr/Cargo.toml

```diff
@@ -20,9 +20,17 @@ travis-ci = { repository = "rust-random/rand" }
 appveyor = { repository = "rust-random/rand" }
 
 [dependencies]
-rand = { path = "..", version = "0.7" }
+rand = { path = "..", version = "0.7", default-features = false }
+num-traits = { version = "0.2", default-features = false, features = ["libm"] }
```

Comment:
Nice, this enables us to support `no_std`. Could you please mention this in the commit message?

Reply:
I've updated the PR title, so since we are using squash and merge, it should become a commit message.